deployments/charts/kuma/values.yaml

```diff
@@ -664,6 +664,13 @@ hooks:
   containerSecurityContext:
     readOnlyRootFilesystem: true
 
+  # -- ebpf-cleanup hook needs write access to the root filesystem to clean ebpf programs
+  ebpfCleanup:
+    # -- Security context at the pod level for crd/webhook/cleanup-ebpf
+    podSecurityContext: {}
+    # -- Security context at the container level for crd/webhook/cleanup-ebpf
+    containerSecurityContext: {}
```

Comment:
Maybe set `readOnlyRootFilesystem: false` to be explicit?

Reply:
Alternatively, I wonder if we should allow setting `readOnlyRootFilesystem: true` at all? I mean it will break without it, right? So maybe we should _force_ it using something like `merge`:

https://github.com/kumahq/kuma/pull/6177/files#diff-f5e236f15e3f80a40ec7e3113b047e692e7b21bca667f61f53f6aea246ad6343L90

Reply:
> Maybe set `readOnlyRootFilesystem: false` to be explicit?

done